editor/store/actions.js

```diff
@@ -353,9 +353,17 @@ export function editPost( edits ) {
 	};
 }
 
-export function savePost() {
+/**
+ * Returns an action object to save the post.
+ *
+ * @param {Object} options Set autosave: true for an autosave.
```

Comment:
Minor: JSDoc has a solution for documenting properties of an argument that we could leverage:

http://usejsdoc.org/tags-param.html#parameters-with-properties

Reply:
nice, changed in https://github.com/WordPress/gutenberg/pull/6257/commits/9142591bfba6ba3fdfb4ea81dc57e3dcb7024504